<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdStartupWindow</class>
 <widget class="QMainWindow" name="QxrdStartupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="m_Layout">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="m_Icon">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTextBrowser" name="m_Messages">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>10</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="m_Description">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_FileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFile"/>
   </widget>
   <widget class="QMenu" name="m_EditMenu">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="m_WindowMenu">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionWindow"/>
   </widget>
   <addaction name="m_FileMenu"/>
   <addaction name="m_EditMenu"/>
   <addaction name="m_WindowMenu"/>
  </widget>
  <action name="actionFile">
   <property name="text">
    <string>File...</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit...</string>
   </property>
  </action>
  <action name="actionWindow">
   <property name="text">
    <string>Window...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
